packages/core/src/components/html-table/_overrides.scss

```
@@ -0,0 +1,81 @@
+// Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+@import "../../common/variables";
+@import "../../common/_customs";
+
+
+
+table.#{$ns}-html-table {
+
+  background-color: $g-light-base3;
+
+  thead tr {
+    box-shadow: inset 0 (-$table-border-width)  0 0 $table-border-color;
+  }
+
+  tfoot {
+    box-shadow: inset 0 ($table-border-width)  0 0 $table-border-color;
```

Footer line is only 1px width instead of 2px:
```suggestion
    box-shadow: inset 0 (2 * $table-border-width)  0 0 $table-border-color;
```